eos/b-decays/observables.cc

```diff
@@ -80,6 +80,18 @@ namespace eos
                         std::make_tuple("q2", "k2"),
                         Options{ { "l", "mu" }, { "lprime", "e" } }),
 
+                make_observable("B_u->taunuee::d2BR/dq2/dk2", R"(\frac{d\mathcal{B}(B^- \to \tau^-\bar\nue^+e^-)}{dq^2/dk^2})",
```

Comment:
Merge the changes to the allowed options into this commit please!

Reply:
This is not valid LaTeX: ``\nu e^+``.

Reply:
Maybe also fix the existing LaTeX errors in e.g. A_FB (but in a separate commit).